app/controllers/verify/finance_controller.rb

```diff
@@ -31,7 +31,7 @@ def step_name
     end
 
     def confirm_step_needed
-      redirect_to verify_address_path if idv_session.financials_confirmation == true
+      redirect_to redirect_url if idv_session.financials_confirmation == true
```

Comment:
Ok, I understand your point now. We want to err on the side of presenting more information about the process to the user, so I'll add the page back and remove the mail verification option from the view.